### ISSUE TITLE
CLI as API driver

### ISSUE THREAD

**Issue by akx** [valohai/valohai-cli#58]
When using the CLI as an API driver, it would be useful to be able to say

```
$ vh --project=UUID exec run ...
```
or maybe even
```
$ vh --project=foo/bar exec run ...
```

to avoid having to link a project to a dummy path and so on.

This would require some additional wrangling so the client knows to fetch the commit config from the remote instead of blindly looking for a YAML file.

In addition, the API token and host should be also readable from environment variables or command line args (which can be neatly done in one fell swoop in Click) to avoid having an interactive login first..

**Comment by akx** [comment]
This will be required for @JuhaKiili 's notebook work too, I bet.

**Comment by akx** [comment]
Shipped in v0.9.0.
